### PR TITLE
Make options options

### PR DESCRIPTION
**Pull request by orakley**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -78,7 +77,7 @@ const config = {
         {
             name: 'Hausnummer',
             identifier: 'housenumber',
-            type: 'Number',
+            type: 'number',
```
Comment (oliverborner) [inline src/App.vue:80]: just for your information: JS refers to Number and TS to number (without single quotes) if you want to display real data types and not string literals

Comment (oliverborner) [reply src/App.vue:80]: could be a huge improvement to make the query builder component type save...maybe in version 5.0

**Review by oliverborner — APPROVED** [review]
LGTM!

File: src/App.vue (hunk 1 above)
Comment (orakley) [reply src/App.vue:80]: would be great if you give me hints how to do this in the future ✌

**Pull request merged**